Tools/jit/_targets.py

```diff
@@ -137,7 +137,15 @@ async def _compile(
             f"-I{CPYTHON / 'Include' / 'internal' / 'mimalloc'}",
             f"-I{CPYTHON / 'Python'}",
             f"-I{CPYTHON / 'Tools' / 'jit'}",
-            "-O3",
+            # -O2 and -O3 include some optimizations that make sense for
```

Comment:
Did you investigate `-Oz` as well? The clang docs are fairly vague, but they say it reduces code size even further, so I'm curious if it's worth investigating as well.

Reply:
Yep, `-Oz` is about 1-2% slower across the board.